llama-index-legacy/llama_index/legacy/readers/json.py

```diff
@@ -84,7 +84,9 @@ def __init__(
         self.ensure_ascii = ensure_ascii
         self.is_jsonl = is_jsonl
 
-    def load_data(self, input_file: str) -> List[Document]:
+    def load_data(
```

Comment:
@thegunslingers could you revert the changes to the legacy folder.
the legacy folder will be deprecated

Reply:
Reverted changes to legacy JSONReader